wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1318,13 +1315,18 @@ class SwiftGenerator private constructor(
         for (type in types) {
           val protoType = type.type
           val name = protoType.simpleName
+
           val className = if (enclosingClassName != null) {
             // Temporary work around for https://bugs.swift.org/browse/SR-13160.
-            enclosingClassName.nestedType(if (name == "Type") "Type_" else name)
+            val safeName = if (name == "Type") "Type_" else name
+
+            enclosingClassName.nestedType(safeName, alwaysQualify = true)
```

Comment:
This is the only functional change. I needed this to fix the code doing `case 1: NestedRequired = try _reader.decode(NestedRequired.self)`